### ISSUE TITLE
make ALT button visible

### ISSUE THREAD

**Issue by QuestioningEspecialy** [syeopite/priviblur#40]
just checked and noticed an image's ALT text is within the image (and therefore copyable if the image is highlighted), but the button to _view_ is not visible
would be nice to have it clickable in some way (and stay up even when scrolling or the selecting part of the ALT text itself)
in short, there should be a ALT button on images that contain ALT text, and it should function better than it does on tumblr
